optuna_dashboard/ts/types/index.d.ts

```diff
@@ -206,3 +208,11 @@ type StudyDetails = {
 type StudyParamImportance = {
   [study_id: string]: ParamImportance[][]
 }
+
+type PreferenceChoice = {
+  uuid: string
```

Comment:
I think `preference_id` is missing here. Also could you rename uuid to `history_id` or `preference_history_id`?

```suggestion
  preference_history_id: string
  preference_id: string
```